README.md

```diff
@@ -1,3 +1,51 @@
 # receipe_app
 
-A new Flutter project.
+How to contribute to this open source project
+ Prerequisites
+* You will need to have flutter set up on your system  [https://docs.flutter.dev/get-started/install]
+* You Should know this project was build using stacked as a statemanagement tool  check this for info on stacked[https://stacked.filledstacks.com/docs/getting-started/overview/]
+ 
+Installation
+
+To get started with the code follow this steps
+
+1. Fork my repo 
+
+...
+click the fork button on the top right corner and give it a name you want
+...
+2. clone your fork
+
+.....//TODO
+git clone https://github.com/DreXuri/recipe_app.git
+.....
+
+2. Navigate to the project directory
+
+....//TODO
+cd recipe_app
+.....
+
+3. clean the app by running the flutter clean
+
+....
+flutter clean
+....
+
+4. run the app
+
+....
+flutter run 
+....
+
+5. make changes
+
+6. create a branch
+
+7. cheackout to your branck
```

Comment:
kindly check the spelling of your `branch`.

Reply:
ok, I'll correct an revert

Reply:
i have addressed your comment